app/src/main/res/values/colors.xml

```diff
@@ -2,9 +2,14 @@
 <resources>
 
     <!-- Main colors -->
-    <color name="colorPrimary">#FF2F343D</color>
-    <color name="colorPrimaryDark">#FF2F343D</color>
+    <color name="colorPrimaryDark">#FF212121</color>
+    <!-- Widechat colors -->
+    <!-- <color name="colorPrimary">#FF303030</color> --> 
+    <color name="colorPrimary">#008085</color>
+    <color name="widechatStatusBar">#0E7075</color> <!-- Widechat color -->
+    <!-- End Widchat Colors -->
     <color name="colorAccent">#FF1D74F5</color>
+    <color name="widechat_fab">#00C1B7</color>
```

Comment:
- line 10 "Widchat"
- might as well put "widechat_fab" in Widechat colors section?